tests/e2e/etcd_mix_versions_test.go

```diff
@@ -73,11 +73,11 @@ func mixVersionsSnapshotTest(t *testing.T, clusterVersion, newInstanceVersion e2
 		e2e.WithVersion(clusterVersion),
 	)
 	if err != nil {
-		t.Fatalf("failed to start etcd cluster: %v", err)
+		require.NoError(t, err, "failed to start etcd cluster: %v", err)
 	}
```

Comment:
Please change 
```
if err != nil {
	require.NoError(t, err, "failed to start etcd cluster: %v", err)
}
```
to
```
require.NoError(t, err, "failed to start etcd cluster: %v", err)
```

Note that this comment applies to the following couple of places.

Reply:
@ahrtr done